fairmq/FairMQParts.h

```diff
@@ -57,6 +52,23 @@ class FairMQParts
         fParts.push_back(std::move(msg));
     }
 
+    /// Add variable list of parts to the container (move)
+    template <typename... Ts>
+    void AddPart(std::unique_ptr<FairMQMessage>&& first, Ts&&... remaining)
+    {
+        AddPart(std::forward<std::unique_ptr<FairMQMessage>>(first));
```

Comment:
this needs to be `std::move` instead of `forward`. `first` is an rvalue ref, `remaining` are actual forwarding references.

Reply:
ah true, because it is not a templated parameter, will fix. Thanks for spotting it